apps/src/music/player/MusicPlayer.ts

```diff
@@ -36,6 +36,7 @@ interface SoundEvent extends PlaybackEvent {
   type: 'sound';
   id: string;
   skipContext?: SkipContext;
+  effects?: {};
```

Comment:
(as discussed offline) we can probably type this more explicitly

Reply:
Will do, thanks!